src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -738,6 +740,39 @@ function validate_connection_args(user_input, action) {
     }
 }
 
+////////////////////////////
+//// UPGRADE VALIDATION ////
+///////////////////////////
+
+/**
+ * validate_expected_version checks that expected_version is valid
+ * if it's not valid it throws an error
+ * @param {string} expected_version
+ */
+function validate_expected_version(expected_version) {
```

Comment:
can we move this method also to new util file?

Reply:
It's about an argument, the `versions_utils.js` is more general (like a library that we can use).